cmd/backup-operator/README.md

```diff
@@ -0,0 +1,104 @@
+# etcd backup operator
+
+## Overview
+
+etcd backup operator backups the data of a etcd cluster running on [Kubernetes][Kube] to a remote storage such as AWS [S3][s3]. Currently, backup operator only supports saving backup to AWS S3.
+
+## Requirements
+* Kubernetes 1.7+
+* etcd 3.1+
+
+## Getting Started
+
+You can try out etcd backup operator by running it on `Kubernetes` and then create a `Backup Custom Resource` which contains the targeting etcd cluster and S3 backup config; the etcd backup operator automatically picks up the `Backup Custom Resource`, retrieves etcd data, and then saves data as a backup to S3.
+>Note: The demo uses the `default` kubernete `namespace`.
+
+First, you need to have access to a [Kubernetes][Kube] environment either locally via [Minikube][minikube] or any Cloud Providers that provide `Kubernetes` as a service.
+
+Once you have access to a `Kubernetes` setup, you need to create a [etcd operator][etcd_operator] pod in order to deploy an etcd cluster. See [instructions on how to install/uninstall etcd operator][etcd_operator_install] for more detail.
```

Comment:
Again, this is duplicating the etcd operator doc.
Just explicitly mention the assumption here that assuming the etcd cluster from `example/example-etcd-cluster.yaml` has been deployed.